semantic-conventions/syntax.md

```diff
@@ -168,6 +178,26 @@ The following is only valid if `type` is `event`:
   If not specified, the `prefix` is used. If `prefix` is empty (or unspecified),
   `name` is required.
 
+#### Metric Group semantic convention
```

Comment:
I'm missing a statement of the form "A metric group is ...", i.e. what is a metric group conceptually.

Reply:
After some rewording, this section now includes: 

> The metric group semconv is a group where related metric attributes can be defined and then referenced from other `metric` groups using `ref`.

Does it make sense?

Reply:
Would this still be needed after #124, or would these be redundant?

Reply:
@jamesmoessis @lmolkova #124 should be unrelated, right?
One is for grouping metrics (to have an identifier to print out related ones in one table) and the other is for attribute groups (so they can be shared as a whole between different signals).

Reply:
#124 can be used for sharing as a whole or part by part. It can also be used to group related attributes and a table can be generated from it on its own IIUC.

Reply:
For grouping entire metric definitions - consisting of generic fields (including attributes) and the metric-specific fields metric_name, instrument and unit - the metric_group would be used.

So I think for the combined table generation that will be done in a follow-up PR as per the thread in https://github.com/open-telemetry/build-tools/pull/79/files#r996803040 this would lay the foundation.